src/components/CommentList/CommentList.scss

```diff
@@ -1 +1,7 @@
-//
```

Comment:
Please replace the line comment (//) with a block comment (/* */) to follow the given rules.